src/msw/graphicsd2d.cpp

```diff
@@ -2704,6 +2710,17 @@ wxCOMPtr<IDWriteTextLayout> wxD2DFontData::CreateTextLayout(const wxString& text
         textLayout->SetStrikethrough(true, textRange);
     }
 
+    if (m_spacing)
+    {
+#if WINVER >= 0x602
+        if (wxGetWinVersion() > wxWinVersion_8)
+        {
+            wxCOMPtr<IDWriteTextLayout1> textLayout1 = reinterpret_cast<wxCOMPtr<IDWriteTextLayout1>>(textLayout);

```

Comment:
@vadz I'm not sure how to convert between the old and new COM interface here, should I use dynamic_cast here?

Reply:
Casts in COM are done using `QueryInterface()`, so you need to ask the existing pointer whether it supports `IDWriteTextLayout1` (it should if you use `IDWriteFactory1` to create it).

It would be nice if `wxCOMPtr` did it automatically for you, but it doesn't currently, so you need to call it manually.